runhouse/resources/hardware/cluster.py

```diff
@@ -1645,3 +1658,44 @@ def _check_for_child_configs(cls, config: dict):
             config["default_env"] = default_env
 
         return config
+
+    def _is_den_authed_and_saved_to_den(self, change_type: str):
```

Comment:
wondering if this check is necessary, is there any harm in just updating the config with the interval field regardless if status check is explicitly enabled or disabled?

Reply:
@jlewitt1 Removed the check if the cluster is saved into den. If it is not, the post request will raise an error, which we catch and send an error log, explaining that the cluster is not saved and therefore we can't post its status.

If left the check of den_auth == True because: 
1. It is not pinging den, therefore it is a "cheap" check
2. I think it is in our interests to clarify to the user that this is a feature requires Den auth / Den user. wdyt?